src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/attr/Change.java

```diff
@@ -144,6 +147,16 @@ private Object readResolve() {
         return this;
     }
 
+    /**
+     * Is this change a Work in progress.
+     */
+    private boolean wip;
+
+    /**
+     * Is this change private.
+     */
+    private boolean isPrivate;
```

Comment:
doesn't checkstyle or findbugs have an issue with this name?

Reply:
Yeah, I don't like the name either :) Understandable that you can't use `private` since that's reserved. But I guess `_private` in that case would be better?